### PR TITLE
{container} Bugfix #15856: az container exec - remove eol check to avoid closing terminal before it even started on linux

### PR DESCRIPTION
**Pull request by bitcloud**
The backend sends an empty webSocket message after establishing an connection. The removed lines of code interpreted that as an EOL which exits the webSocket read loop.
As the Windows code doesn't do that as well, I removed it here as well now.
The terminal gets closed after the webSocket closes anyway.

Fixes: https://github.com/Azure/azure-cli/issues/15856

**Description**
As described in https://github.com/Azure/azure-cli/issues/15856 there is currently an issue getting a remote terminal to a container instance with `az container exec` as it immediately closes the connection.

**Testing Guide**
- start a container e.g. with `az container create` with a long living container
- try to execute a shell inside the container with `az container exec --exec-command "/bin/bash" ...`
- currently it just exits without an error or shell

**History Notes**
[container] az container exec: remove eol check to avoid closing terminal before it even started on linux

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by bitcloud** [comment]
I'm not quite sure if I should submit that as a hotfix as well to merge that also against the current version?

**Comment by yonzhan** [comment]
container

**Review by qwordy — APPROVED** [review]

**Comment by bitcloud** [comment]
Any updates here? Anything else I should do?

**Comment by qwordy** [comment]
> 
> 
> Any updates here? Anything else I should do?

Wait for code owner to review.

**Comment by qwordy** [comment]
@zhoxing-ms

**Comment by zhoxing-ms** [comment]
Actually, I am not the owner of container module, which is responsibility for service team
@hjhhh3000vitae Could you please help to review this PR or ask the relevant person to look at it?

**Comment by hjhhh3000vitae** [comment]
Sorry I'm not the owner either and I am not aware which team is the owner.

**Review by qwordy — APPROVED** [review]

**Comment by qwordy** [comment]
@zhoxing-ms Are you willing to be owner of this module?

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**
